taker/src/main.rs

```diff
@@ -346,6 +352,11 @@ async fn main() -> Result<()> {
         }
     };
 
+    let ext_priv_key = match opts.wallet_xprv {
+        Some(wallet_xprv) => wallet_xprv,
+        None => ext_priv_key,
+    };
```

Comment:
🔧 I think it would be clearer if we had a

```rust
match (opts.umbrel_seed, opts.wallet_xprv) {
// handle combinations
}
```

handling all the cases explicitly.

And I would expect it to be mutually exclusive to the Umbrel seed. But I'm not sure how to express that in the CLI.

Reply:
The way it is is the least invasive in regards to new lines of code 😬

I thought about the xor between `umbrel_seed` and `wallet_xprv`. The thing is, that the password is derived from the `umbrel_seed`, unless we go password less that is our only option right now.

Reply:
Actually this is not an issue because we control umbrel setup 😃